R/viz.R

```diff
@@ -658,180 +810,341 @@ pretty_lollipop_plot = function(maf_df,
 #' #get ashm regions
 #' some_regions = grch37_ashm_regions
```

Comment:
I suspect this example is out of date. Do we need to call the new function to get the regions in the right format?